TermiFlux_TUI.sh

```
@@ -0,0 +1,227 @@
+    3)
+        clear
+        echo -e "\033[1;32mThank you for using TermiFlux!\033[0m"
+        exit 0
+        ;;
+    *)
+        clear
+        echo -e "\033[1;31mNo selection made. Exiting...\033[0m"
+        exit 1
+        ;;
+esac
+EOFcat > TermiFlux_TUI.sh << 'EOF'
```

Shellcheck (0.11.0)

[error] Couldn't parse this here document. Fix to allow more checks.

(SC1073)